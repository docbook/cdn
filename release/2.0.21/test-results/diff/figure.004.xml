<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <figure deltaxml:deltaV2="A" id="figduck" class="figure-wrapper">
            <div class="figure">
               <div id="figduck.2" class="mediaobject centerimg">
                  <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
                  <img src="../graphics/duck.png" align="middle" alt="The DocBook: TDG Duck" longdesc="ld-figduck.2.html"/>
               </div>
            </div>
            <div class="title">Figure 1. The Duck</div>
         </figure><div deltaxml:deltaV2="B" class="content">
            <figure id="figduck" class="figure-wrapper">
               <div class="figure">
                  <div id="figduck.2" class="mediaobject centerimg">
                     <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
                     <img src="../graphics/duck.png" align="middle" alt="The DocBook: TDG Duck" longdesc="ld-figduck.2.html"/>
                  </div>
               </div>
               <div class="title">Figure 1. The Duck</div>
            </figure>
         </div>
         <div deltaxml:deltaV2="A=B" class="longdesc-list">
            <div id="longdesc-xxx" class="dialog longdesc-hide">
               <p>A woodcut from the Dover Archives, this wood
duck is used as the “animal” for <em class="citetitle">DocBook: The Definitive
Guide</em>.</p>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="../../resources/js/annotation.js"/>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>