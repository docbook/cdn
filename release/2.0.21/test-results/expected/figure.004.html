<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><figure id="figduck" class="figure-wrapper"><div class="figure"><div id="figduck.2" class="mediaobject centerimg"><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d10935e32" title="long description link">D</a>]</div><img src="../graphics/duck.png" align="middle" alt="The DocBook: TDG Duck" longdesc="ld-figduck.2.html" /></div></div><div class="title">Figure 1. The Duck</div></figure><div class="longdesc-list"><div id="longdesc-d10935e32" class="dialog longdesc-hide"><p>A woodcut from the Dover Archives, this wood
duck is used as the “animal” for <em class="citetitle">DocBook: The Definitive
Guide</em>.</p></div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></body></html>